lanuv_download/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:latest 
+
+# install dependencies
+RUN apk update && \
+    apk del sed && \

```

Comment:
why is `sed` removed and then installed again?

Reply:
Alpine docker base image comes with some limited busybox-sed-implementation. Remove and install fully-fledged sed package for parser function.

Reply:
alright,  I did not know that